src/main/java/de/uos/informatik/ko/rcp/geneticalgorithm/GeneticAlgorithm.java

```diff
@@ -31,6 +32,13 @@
         // Only for debug purposes
         var updateDeltas = new LinkedHashMap<Long, Integer>(); // <update delta, new makespan>
 
+        int[] makespans = new int[popsize]; // makespans ist mit 0en gefüllt, wenn wir es nicht benutzen, sonst wird es initialisiert
+
+        boolean makeSpans = false;
+        if(Config.instance().cacheMakespans){
+            makeSpans = true;
+        }
+
         // bestimme Wkeit (Wert zwischen 0 und 1) dass eine Mutation auftritt
         final double mutationswkeit = 0.4;
```

Comment:
I think it would make sense ot use Config.mutationProbability here.

Reply:
We wanted the branches to have one purpose only and did not change several aspects at one time. By merging the branches into the master, we will achieve the desired functionality